frontend/packages/component-library/src/actionBlock/ActionBlock.tsx

```diff
@@ -0,0 +1,235 @@
+import classNames from 'classnames';
+import {HTMLAttributes} from 'react';
+
+import {LinkButton} from '@/button';
+import {
+  Heading3,
+  BodyThreeText,
+  OverlineTwoText,
+  StrongText,
+} from '@/typography';
+
+import moduleStyles from './actionBlock.module.scss';
+
+export interface ActionBlockProps extends HTMLAttributes<HTMLDivElement> {
+  /** Action Block title */
+  title: string;
+  /** Action Block description */
+  description: string;
+  /** Action Block image */
+  image?: string;
```

Comment:
I see that this component is part of general design system. I wonder if we should make our `Image` component part of general design system as well and use it here (like we currently do in `TabGroup`, for example). This would make dsco more consistent and easier to use I believe.

cc @moshebaricdo what do you think of it from design point of view?

Am I missing something?

This might as well be a follow up, of course.

Same as with button props, I think it migth be worth using full ImageProps type here for to make it easier and more consistent and then render image component
```suggestion
  image?: ImageProps;
```

Reply:
Hey Denys! I do like this idea, but I wonder if it makes sense to do this as a follow up improvement? Currently we need the fixed dimensions for two specific components (action block and tab group), and it sounds like refactor of the image to work this way is tricky. Maybe it makes sense to just implement a solution for tab group and action block for now to save some time?

Reply:
I can try to make this work using an `<img>` tag, but I agree that changes to the Image component should be a followup.